pages/interface/components/Header/index.js

```diff
@@ -77,6 +78,17 @@ export default function HeaderComponent() {
 
       {user && (
         <>
+          <PrimerHeader.Item
+            sx={{
+              mr: 2,
+            }}>
+            <Tooltip aria-label="Publicar novo conteúdo" direction="s" noDelay={true} wrap={true}>
+              <HeaderLink href="/publicar" aria-label="Publicar novo conteúdo">
```

Comment:
```suggestion
              <HeaderLink href="/publicar">
```

Estou em dúvida se precisamos mesmo duplicar o `aria-label` que já está presente no `Tooltip`?